wordpress-mobile/WordPress-iOS
# Removing FakePreviewBuilder

Now that we are auto saving a post and uploading a draft before preview,
we can take the last step in to get rid of local previews and remove the FakePreviewBuilder from the code. The expected behavior is to show an "error view" when ever a preview request fails instead of showing the fake preview. 

In this PR we are replacing the calls to showFakePreview() with previewFailed delegate method while logging the reason for the failure to Sentry.

This failure is unlikely to happen since we added a check for the existence of the post permalink before generating the preview.

**Before**

![preview_offline_3](https://user-images.githubusercontent.com/1335657/57958081-6b870280-78b3-11e9-8a6e-0d8c45dcb718.gif)

**after**
![preview_offline_2](https://user-images.githubusercontent.com/1335657/57958178-c02a7d80-78b3-11e9-8f62-9034817adc30.gif)


Fixes #11699 

To test:
1. Be offline
2. Create a new draft
3. preview the draft
4. See the "Empty view"

Update release notes:
